ssat-api.sh

```diff
@@ -233,50 +233,54 @@ function callTestExitEq() {
 function callCompareTest() {
 	# Try cmp.
 	output=0
-	command -v cmp
-	# If cmp is symlink, then it could be from busybox and it does not support "-n" option
-	if [[ $? == 0 && ! -L $(which cmp) ]]
-	then
-		# use cmp
-		if (( $5 == 0 )); then
-			# Size should be same as well.
-			cmp $1 $2
-			output=$?
-		elif (( $5 == 1 )); then
-			# Compare up to the size of golden
-			cmp -n `${StatCmd_GetSize} $1` $1 $2
-			output=$?
-		elif (( $5 == 2 )); then
-			# Compare up to the size of test-run
-			cmp -n `${StatCmd_GetSize} $2` $1 $2
-			output=$?
+	if [[ ! -f "$1" || ! -f "$2" ]]; then
+		testResult $output "$3" "$4" $6
```

Comment:
please do return in ```then``` case so that you don't need to add ```else``` case and indent further.

Reply:
Agreed with https://github.com/myungjoo/SSAT/pull/15/files#r501473366.

@myungjoo Q: Which repository is the upstream for SSAT? github or git.tzien.org?

Reply:
Updated, please check!

Reply:
github.com/myungjoo/SSAT is upstream and others are forks.

Reply:
Anyway, don't do SR until 6.0 M2 is released.